turbo.json

```diff
@@ -1,19 +1,17 @@
 {
   "$schema": "https://turbo.build/schema.json",
   "pipeline": {
-    "test": {
+    "test:unit": {
       "inputs": [
         "{components,hooks,layouts,providers,types,util}/**/*.{ts,tsx}",
         "jest.config.mjs",
         "jest.setup.mjs"
       ],
       "outputMode": "full"
     },
-    "test:ci": {
+    "test:storybook": {
       "inputs": [
-        "{components,hooks,layouts,providers,types,util}/**/*.{ts,tsx}",
-        "jest.config.mjs",
-        "jest.setup.mjs"
+        "{components,hooks,layouts,providers,types,util}/**/*.{ts,tsx}"
```

Comment:
Storybook doesn't use the regular jest config file I suppose?

Reply:
It doesn't